Script/AtomicEditor/ui/frames/inspector/ComponentInspector.ts

```diff
@@ -211,7 +211,8 @@ class ComponentInspector extends Atomic.UISection {
 
             // refresh entire inspector, fix this...
             this.sendEvent("EditorActiveNodeChange", { node: node });
-
+            this.sendEvent("ComponentRemoved");
```

Comment:
We shouldn't be generating ComponentAdded/ComponentRemoved etc events like this, there are events being created when Components are created/removed already and this will get really confusing